### PR TITLE
[KOGITO-58] Upgrading GraalVM libraries

### PR DESCRIPTION
**Pull request by ricardozanini**
Please see:
https://issues.jboss.org/browse/KOGITO-58

This PR adds a new module 19.x under GraalVM path. I wonder that keeping a *.x could be simpler in the future to upgrade the 19.
There's a lot of enhancements that worth the upgrade, plus the new Quarkus version doesn't work with the old GraalVM.

@spolti @mswiderski could you guys please take a look?

### CODE REVIEW

File: s2i/kogito-quarkus-centos-s2i-overrides.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ modules:
   install:
   - name: add-kogito-user
   - name: graalvm
-    version: "1.0.0-rc16"
+    version: "19.1.0"
```
Comment (mswiderski) [inline s2i/kogito-quarkus-centos-s2i-overrides.yaml]: currently quarkus only supports 19.0.2 and not 19.1.0 so I'd stay with 19.0.2 for now

Comment (ricardozanini) [reply s2i/kogito-quarkus-centos-s2i-overrides.yaml]: downgraded!

**Review by mswiderski — APPROVED** [review]

**Comment by mswiderski** [comment]
let's wait for @spolti review and merge, thanks @ricardozanini

**Review by spolti — APPROVED** [review]

**Comment by spolti** [comment]
@mswiderski I'll update the images tomorrow morning, just need to confirm the tag to use, can I tag as 0.2.0 then delete the previous one?

**Comment by spolti** [comment]
Right, just saw your comment on the Jira.

**Comment by mswiderski** [comment]
yes, 0.2.0 makes sense and then remove the old ones. thanks

**Pull request merged**